schema/gen/go/adjunctCfg.go

```diff
@@ -65,6 +65,11 @@ func (cfg *AdjunctCfg) FieldSymbolUpper(f schema.StructField) string {
 	return strings.Title(f.Name())
 }
 
+// Comments returns a bool for whether comments should be included in gen output or not.
+func (cfg *AdjunctCfg) Comments() bool {
+	return true // FUTURE: okay, maybe this should be configurable :)
+}
```

Comment:
Do we really need this? It's trivial to remove comments from Go source code by just using `go/parser` with the comment flag disabled, and re-printing the program. Or, if the code is simple enough, a bit of sed.